<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QLabel" name="Notes">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Modern</family>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(225, 225, 225); font-size:28pt;</string>
   </property>
   <property name="text">
    <string>   Notes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(55, 57, 55); font-size:14pt; color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="notesBtn">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>20</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(55, 57, 55); font-size:14pt; color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Take Notes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="notesViewBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(55, 57, 55); font-size:14pt; color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>View All Notes</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
